sphinx_rtd_theme/static/js/versions.js_t

```diff
@@ -123,3 +124,68 @@ function renderLanguages(config) {
       document.dispatchEvent(event);
     });
  });
+{%- endif %}
+
+
+{%- if theme_version_selector or theme_language_selector %}
+function onSelectorSwitch(event) {
+     const option = event.target.selectedIndex;
+     const item = event.target.options[option];
+     window.location.href = item.dataset.url;
+ }
+
+document.addEventListener("readthedocs-addons-data-ready", function(event) {
+  const config = event.detail.data();
+
+  const dropdownSelectors = document.querySelector("div.dropdown-selectors");
+
+  {%- if theme_version_selector %}
+  const versionSelect = `
+    <select id="readthedocs-version-selector">
+      ${ config.versions.active.map(
+          (version) => `
+          <option
+              value="${ version.slug }"
+              ${ config.versions.current.slug === version.slug ? 'selected="selected"' : '' }
```

Comment:
A couple things this needs to support:

- Either don't show this menu on PR builds or show this menu with the PR build as the selected version, so that there isn't a conflicting menu on PR builds

![image](https://github.com/user-attachments/assets/5755042e-cb3a-492d-af51-41a319a5cb3f)

- Default to the default branch. Otherwise it just shows the first option (`0.5.0` in this case, which is not good UX)

Reply:
> show this menu with the PR build as the selected version

I followed this approach.